src/openeo_grass_gis_driver/collections.py

```diff
@@ -89,5 +91,33 @@ def get(self):
                     #                       location, mapset)))
                     #     COLLECTIONS_LIST.append(ds)
 
+            # Additionally check for STAC collections registered in actinia
+            status_code, stac_collections = self.iface.get_stac_collections()
+            if status_code != 200:
+                # log.warning("Couldn't get STAC collections from actinia")
+                stac_collections = []
+
+            for i in stac_collections['collections']:
+                try:
+                    title = i['title']
+                except Exception:
+                    title = i['id']
+                try:
+                    license = i['license']
+                except Exception:
+                    license = "proprietary"
```

Comment:
Or unknown? You decide.

Reply:
"proprietary" is currently [the default](https://github.com/Open-EO/openeo-grassgis-driver/blob/master/src/openeo_grass_gis_driver/collections.py#L66) in the existing implementation, so I would rather not change it.